setup.py

```diff
@@ -59,13 +59,13 @@
         ]
 
 DEPENDENCIES = [
-        "numpy",
-        "pandas",
-        "pandas-datareader>=0.1.1",
-        "matplotlib",
-        "lxml", 
+        "numpy=1.10.4",
```

Comment:
Maybe we shouldn't change this after all. I don't think it's a good idea to require that users install a single exact version of each library. Let me build a virtual env to test all this myself. The one case where we might want to update the dependency is pandas-datareader. That's the one where versions created problems.